kubetest/gke.go

```diff
@@ -347,7 +353,7 @@ func (g *gkeDeployer) getKubeConfig() error {
 	}
 	if err := finishRunning(exec.Command("gcloud", "container", "clusters", "get-credentials", g.cluster,
 		"--project="+g.project,
-		"--zone="+g.zone)); err != nil {
+		g.location)); err != nil {
```

Comment:
Yeah, I realized that shortly after pushing this one. See #3967 (and lgtm if you feel like it :))